PolicyServicePkg/Test/PolicyServicePkgHostTest.dsc

```diff
@@ -12,7 +12,7 @@
   PLATFORM_VERSION        = 0.1
   DSC_SPECIFICATION       = 0x00010005
   OUTPUT_DIRECTORY        = Build/PolicyServicePkg/HostTest
-  SUPPORTED_ARCHITECTURES = IA32|X64|AARCH64
+  SUPPORTED_ARCHITECTURES = IA32|X64

```

Comment:
Similarly we could merge this. Co-authored-by can be added if you want to preserve any authorship history